README.md

```diff
@@ -76,17 +76,13 @@ More details about API versioning and status in Kubernetes can be found on the K
-| YugabyteDB  | [YugabyteDB](https://docs.yugabyte.com/latest/introduction/) is a high-performance, cloud-native distributed SQL database which can tolerate disk, node, zone and region failures automatically.                                                                                                                       | yugabytedb.rook.io/v1alpha1  | [Deprecated](https://github.com/rook/rook/issues/6992#issuecomment-771297708)* |
-
-\* CockroachDB, EdgeFS, and YugabyteDB were removed from Rook in v1.6. See [Rook v1.5](https://rook.github.io/docs/rook/v1.5/) docs if still interested.
-   For YugabyteDB, see the replacement operator [here](https://github.com/yugabyte/yugabyte-operator).
+| Name                                           | Details                                                                                                                                                                                                                                                                                                                | API Group                  | Status |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------- | ------ |
+| Ceph                                           | [Ceph](https://ceph.com/) is a distributed storage system that provides file, block and object storage and is deployed in large scale production clusters.                                                                                                                                                             | ceph.rook.io/v1            | Stable |
+| [Cassandra](https://github.com/rook/cassandra) | [Cassandra](http://cassandra.apache.org/) is a highly available NoSQL database featuring lightning fast performance, tunable consistency and massive scalability. [Scylla](https://www.scylladb.com) is a close-to-the-hardware rewrite of Cassandra in C++, which enables much lower latencies and higher throughput. | cassandra.rook.io/v1alpha1 | Alpha  |
+| [NFS](https://github.com/rook/nfs)             | [Network File System (NFS)](https://github.com/nfs-ganesha/nfs-ganesha/wiki) allows remote hosts to mount file systems over a network and interact with those file systems as though they are mounted locally.                                                                                                         | nfs.rook.io/v1alpha1       | Alpha  |
```

Comment:
Should we remove these items and just stick with the links below to their main repos?

Reply:
Agreed, with the links below, we can just have the status on their own readme.